assembl/views/__init__.py

```diff
@@ -577,7 +580,10 @@ def get_topic(request):
         else:
             topic_dict = topic_dict["titleEntries"]
             locale = discussion.preferences["preferred_locales"][0]
-            return adapt_to_html_content(topic_dict.get(opengraph_locale, topic_dict[locale]))
+            if locale in topic_dict:
+                return adapt_to_html_content(topic_dict.get(opengraph_locale, topic_dict[locale]))
+            else:
+                return "No topic available in the extra json."
```

Comment:
No no, try to pass whatever the original language's description is. And if there is None, you return empty string. Apply to the other scenario as well.